packages/drivers/odsp-driver/src/odspDocumentDeltaConnection.ts

```diff
@@ -296,8 +298,11 @@ export class OdspDocumentDeltaConnection extends DocumentDeltaConnection {
 			relayUserAgent: [client.details.environment, ` driverVersion:${pkgVersion}`].join(";"),
 		};
 
+		connectMessage.supportedFeatures = {
+			[feature_submit_signals_v2]: true,
+		};
```

Comment:
Is there a value for client to convey this info? Just curions

Reply:
Yes. Push will handle the submit_signal event differently when this feature is passed. It changes the structure of the incoming message and adds support for the `targetClientId` property